app/models/content_generation_run.rb

```diff
@@ -17,17 +17,17 @@
 #
 
 class ContentGenerationRun < ApplicationRecord
-  has_many :dashboard_alerts
-  has_many :find_out_mores
-  has_many :management_priorities
-  has_many :analysis_pages
-  has_many :management_dashboard_tables
-  has_many :alert_subscription_events
+  has_many :dashboard_alerts, dependent: :delete_all
+  has_many :find_out_mores, dependent: :delete_all
+  has_many :management_priorities, dependent: :delete_all
+  has_many :analysis_pages, dependent: :delete_all
+  has_many :management_dashboard_tables, dependent: :delete_all
+  has_many :alert_subscription_events #, dependent: :delete_all
```

Comment:
AlertSubscriptionEvent does not have a `content_generation_run_id` column - should it or should this line be removed?  (it does have a `alert_type_rating_content_version_id`)

```
irb(main):038:0> AlertSubscriptionEvent
=> AlertSubscriptionEvent(id: integer, alert_id: integer, contact_id: integer, status: integer, communication_type: integer, message: text, created_at: datetime, updated_at: datetime, email_id: integer, alert_type_rating_content_version_id: integer, find_out_more_id: integer, unsubscription_uuid: string, priority: decimal, subscription_generation_run_id: integer)
irb(main):039:0> ManagementDashboardTable
=> ManagementDashboardTable(id: integer, content_generation_run_id: integer, alert_id: integer, alert_type_rating_content_version_id: integer, created_at: datetime, updated_at: datetime)
```

Reply:
That looks like a mistake to me. They belong to a SubscriptionGenerationRun